can_bus/quadruna/VC/VC_tx.json

```diff
@@ -717,5 +717,19 @@
                 "unit": "L/H"
             }
         }
+    },
+    "Efuse_Faults": {
+        "msg_id": 226,
+        "cycle_time": 100,
+        "description": "VC efuse retry protocol faults",
+        "signals": {
+            "Efuse": {
+                "enum": "Efuse_Enum"
+            },
+            "Fault_Count": {
+                "bits": 3,
+                "signed": false
+            }
+        }
     }
```

Comment:
nitpicks:
`Efuse_Faults` -> `EfuseFaults` (Pascal case)
`Efuse` -> `FaultedEfuse` (EFuse is not a very descriptive name)
`Fault_Count` -> `EFuseFaultCount` (Fault count doesn't necessarily refer to the efuses)